src/main/java/de/retest/recheck/review/ignore/MatcherFilter.java

```diff
@@ -53,7 +55,9 @@ public String toString() {
 				Pair.of( AttributeRegexFilter.class, new AttributeRegexFilterLoader() ), //
 				Pair.of( AttributeRegexFilter.class, new LegacyAttributeRegexFilterLoader() ), //
 				Pair.of( PixelDiffFilter.class, new PixelDiffFilterLoader() ), //
-				Pair.of( ValueRegexFilter.class, new ValueRegexFilterLoader() ) //
+				Pair.of( ValueRegexFilter.class, new ValueRegexFilterLoader() ), //
+				Pair.of( InsertedFilter.class, new InsertedFilterLoader() ), //
+				Pair.of( DeletedFilter.class, new DeletedFilterLoader() ) //
```

Comment:
Does it make sense to add this to the attribute? e.g.
```
attribute=border, deleted
```
to ignore changes like
```
key: border, expected: 1px solid red, actual: (default or absent)
```
same for inserted?

Reply:
Interesting. Hadn't thought of that.

Reply:
That can IMHO already be addressed with what we currently have. So I wouldn't add this now, but will remain open should the need arise.